Platform/MeteorlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -445,22 +445,25 @@ UpdateFspConfig (
   Fspmcfg->PchDmiRtlepceb                                       = 0x0;
   Fspmcfg->SkipCpuReplacementCheck                              = 0x1;
   Fspmcfg->CridEnable                                           = 0x0;
-  Fspmcfg->DqsMapCpu2DramMc0Ch0[1] = 0x1;
-  Fspmcfg->DqsMapCpu2DramMc0Ch1[1] = 0x1;
-  Fspmcfg->DqsMapCpu2DramMc0Ch2[1] = 0x1;
-  Fspmcfg->DqsMapCpu2DramMc0Ch3[1] = 0x1;
-  Fspmcfg->DqsMapCpu2DramMc1Ch0[1] = 0x1;
-  Fspmcfg->DqsMapCpu2DramMc1Ch1[1] = 0x1;
-  Fspmcfg->DqsMapCpu2DramMc1Ch2[1] = 0x1;
-  Fspmcfg->DqsMapCpu2DramMc1Ch3[1] = 0x1;
-  Fspmcfg->DmiGen3DsPortRxPreset[0] = 0x1;
-  Fspmcfg->DmiGen3DsPortRxPreset[1] = 0x1;
-  Fspmcfg->DmiGen3DsPortRxPreset[2] = 0x1;
-  Fspmcfg->DmiGen3DsPortRxPreset[3] = 0x1;
-  Fspmcfg->DmiGen3DsPortRxPreset[4] = 0x1;
-  Fspmcfg->DmiGen3DsPortRxPreset[5] = 0x1;
-  Fspmcfg->DmiGen3DsPortRxPreset[6] = 0x1;
-  Fspmcfg->DmiGen3DsPortRxPreset[7] = 0x2;
+  Fspmcfg->DqsMapCpu2DramMc0Ch0[1] = 0x0;
+  Fspmcfg->DqsMapCpu2DramMc0Ch1[1] = 0x0;
+  Fspmcfg->DqsMapCpu2DramMc0Ch2[1] = 0x0;
+  Fspmcfg->DqsMapCpu2DramMc0Ch3[1] = 0x0;
+  Fspmcfg->DqsMapCpu2DramMc1Ch0[1] = 0x0;
+  Fspmcfg->DqsMapCpu2DramMc1Ch1[1] = 0x0;
+  Fspmcfg->DqsMapCpu2DramMc1Ch2[1] = 0x0;
+  Fspmcfg->DqsMapCpu2DramMc1Ch3[1] = 0x0;
+  Fspmcfg->DmiGen3DsPortRxPreset[0] = 0x7;
+  Fspmcfg->DmiGen3DsPortRxPreset[1] = 0x7;
+  Fspmcfg->DmiGen3DsPortRxPreset[2] = 0x7;
+  Fspmcfg->DmiGen3DsPortRxPreset[3] = 0x7;
+  Fspmcfg->DmiGen3DsPortRxPreset[4] = 0x7;
+  Fspmcfg->DmiGen3DsPortRxPreset[5] = 0x7;
+  Fspmcfg->DmiGen3DsPortRxPreset[6] = 0x7;
+  Fspmcfg->DmiGen3DsPortRxPreset[7] = 0x7;
+  Fspmcfg->PchPcieClkSrcClkReq[0] = 0xff;
+  Fspmcfg->PchPcieClkSrcClkReq[1] = 0xff;
```

Comment:
For above UPDs, MTL already defined config data (refer SblOpen\Platform\MeteorlakeBoardPkg\CfgData\CfgData_Memory.yaml). should use config data to update these UPDs instead hardcode them.

Reply:
updated the upds in config data instead of hardcoding in new commit